Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -1898,6 +1891,66 @@ private static OrderType ConvertOrderType(IBApi.Order order)
             }
         }
 
+        /// <summary>
+        /// Maps TimeInForce from IB to LEAN
+        /// </summary>
+        private static TimeInForce ConvertTimeInForce(string timeInForce)
+        {
+            switch (timeInForce)
+            {
+                case IB.TimeInForce.Day:
+                    return TimeInForce.Day;
+
+                //case IB.TimeInForce.GoodTillDate:
+                //    return TimeInForce.GoodTilDate;
+
+                //case IB.TimeInForce.FillOrKill:
+                //    return TimeInForce.FillOrKill;
+
+                //case IB.TimeInForce.ImmediateOrCancel:
+                //    return TimeInForce.ImmediateOrCancel;
+
+                case IB.TimeInForce.MarketOnOpen:
+                case IB.TimeInForce.GoodTillCancel:
+                default:
+                    return TimeInForce.GoodTilCancelled;
+            }
+        }
+
+        /// <summary>
+        /// Maps TimeInForce from LEAN to IB
+        /// </summary>
+        private static string ConvertTimeInForce(Order order)
+        {
+            if (order.Type == OrderType.MarketOnOpen)
+            {
+                return IB.TimeInForce.MarketOnOpen;
+            }
+            if (order.Type == OrderType.MarketOnClose)
+            {
+                return IB.TimeInForce.Day;
+            }
+
+            switch (order.TimeInForce)
+            {
+                case TimeInForce.Day:
+                    return IB.TimeInForce.Day;
+
+                //case TimeInForce.GoodTilDate:
+                //    return IB.TimeInForce.GoodTillDate;
+
+                //case TimeInForce.FillOrKill:
+                //    return IB.TimeInForce.FillOrKill;
+
+                //case TimeInForce.ImmediateOrCancel:
+                //    return IB.TimeInForce.ImmediateOrCancel;
+
+                case TimeInForce.GoodTilCancelled:
+                default:
+                    return IB.TimeInForce.GoodTillCancel;
```

Comment:
Something I'm just realizing. In english 'canceled' and 'cancelled' are both equally correct spellings. We already use 'canceled' with a single L in `OrderStatus.Canceled` -- so let's keep the spelling the same for all our sanity :P